subgraphs/compound-forks/src/mapping.ts

```diff
@@ -1056,6 +1062,93 @@ export function _handleNewReserveFactor(
   market.save();
 }
 
+export function _handleTransfer(
+  event: ethereum.Event,
+  marketID: string,
+  to: Address,
+  from: Address,
+  comptrollerAddr: Address
+): void {
+  let protocol = LendingProtocol.load(comptrollerAddr.toHexString());
+  if (protocol == null) {
```

Comment:
nit: be consistent with the way we null check. either `!xxx` or `xxx == null`

here we `== null` to null check, but later on `fromAccount` and `toAccount` we do `!xxx` :P